### ISSUE TITLE
SRTM15+ vs SYNBATH V1.2

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmtserver-admin#105]
Hi @GenericMappingTools/gmt-contributors and @GenericMappingTools/core: David Sandwell et al will release a new version of the SRTM15+ grid we use to derive @earth_relief_???.grd from.  Normally, we would just get it and rerun the processing (scripts-driven but manually done by me, not auto on server yet [tbd]).  However, there is a complication in that they also release SYNBATH v.1.2.  The difference is that the SYNBATH grid adds two additional synthetic improvements:

1. Newly identified small seamounts are modeled as circular Gaussian shapes and the bathymetric Gaussian of some height,  fixed radius-to-height ratio and density contrast whose VGG anomaly best matches the observed VGG is used to overwrite the regular inversion (which gives too smooth shapes and too low height for these small seamounts).
2. Using the full seafloor roughness machinery of Goff, Jordan etc they model seafloor abyssal hill roughness statistically by matching synthetic abyssal hills using spreading directions from the age grid to VGG anomalies and fill that into areas where there are no bathymetry constraints.

The result of this is that SRTM15+ and SYNBATH differ in areas of no data constraints: SRTM15+ will be smooth in those areas due to the inversion of gravity superimposed on long-wavelength bathymetry only, while SYNBATH will have "anatomically correct" small seamounts of improved amplitude and slope as well as realistic seafloor roughness in younger unmapped seafloor.

The casual user just wanting a relief map will probably be happy with SYNBATH.  Scientists working with the seafloor in any capacity are likely to want both versions, for different uses.  I am in that camp myself, and the convenience of using @earth_relief means I would like to see both versions in GMT.

The question is how we handle this.  Imagine we called this @earth_shape or something better. We are obviously not storing 41 Gb or earth_shape_01s_g.nc files since they are the same as earth_relief_01s_g.nc. It is only in the oceans they will differ. So if the oceans are 70% of Earth then quite a few earth_relief tiles (at various resolutions) will get a corresponding earth_shape tile. The entire earth_relief_** tree directory is 51 Gb on the server.  Not counting the 03s and 01s tiles it is only 3.2 Gb and if 70% is representative then 2.3 Gb.  That is not much, if we can get it to work.

The paper describing these grids are concerned that users, especially the casual ones, will be confused and think the oceans have fully been mapped.  I think these concerns apply to users that only see these data in Google Earth.  GMT is usually used by more science-literate folks, but we would need to be clear in the documentation what the two flavors are.

Let me know your thoughts on this, including a good name that starts with earth_*.

**Comment by leouieda** [comment]
@PaulWessel I agree that it would be nice to have both of them accessible from different file names. The `earth_relief` grids should probably stay SRTM15+. A few thoughts:

1. 41Gb is not that much. Is it worth going through the trouble of setting up and maintaining (!) infrastructure to get different tiles depending on resolution and geographic  location just to save a few Gbs of server drive space? Remember that this will have to be updated at some point (I don't think Dave will stop producing these grids any time soon) so minimizing the human time needed to do this is more important than minimizing storage.
2. In the docs, it would be very helpful to give examples of when people should use one over the other.
3. `earth_relief_synth` ? Naming is hard.

**Comment by PaulWessel** [comment]
I agree 41Gb is not much but suspect @joa-quim will disagree.  Clearly, for simplicity, sustainability, and automation the answer is to just make duplicate files.  Yet, perhaps having an earth_shape_01s (and 03s) directory with ~21000 symbolic links to the same files in earth_relief_01s will take care of 95% of the duplication without anyone needing to do much. I can experiment with that.

**Comment by PaulWessel** [comment]
While at it, perhaps clever folks like @seisman and @meghanrjones can think of how we can automatically notice when a new SRTM grid is released.  I noticed David's paper (I am reading it as the nth co-author) says it is version 2.3 while we are still at 2.1, so perhaps we missed a quiet 2.2 release to the ftp site?  Similar to how we detect new scientific color maps?

**Comment by Esteban82** [comment]
From [README_V2.3.txt](https://topex.ucsd.edu/pub/srtm15_plus/README_V2.3.txt)

```
SRTM15+V2.2 - April 1, 2021

This V2.2 is an update of V2.1. Note that V2.2 was made special for the GEBCO 2021 grid.  It is the same as V2.1 except that
there were about 1000 small edits to clean up the issues with outer beams, etc.  These were done by Pauline Weatheral , David Sandwell and
Brook Toxer. In addition two land tiles were replaced. This grid was never distributed to the public.
```

**Comment by Esteban82** [comment]
I like `earth_relief_synth` for the name. It should be something that clearly indicates that it is not completed real.

**Comment by meghanrjones** [comment]
I agree with Leo in that I do not view 41 Gb as that large. `earth_relief_synth` seems good.

I'll look into developing an SRTM release tracking workflow.

**Comment by leouieda** [comment]
Also, it’s 41gb in the server and only on the users computer if they decide to plot the whole world at this resolution. I like the symlink idea but its also something that will have to de duplicated for every update.

**Comment by leouieda** [comment]
One possible way to check updates is to have a CI job query the server every so often and create an issue if the contents change.

**Comment by PaulWessel** [comment]
My symlink idea would only save space on the server (which is not really an issue).  It would not save user space without us internally changing references to earth_shape_01|3s tiles to earth_relief_01|3s tiles instead.  Perhaps that is not such a crazy idea though. Saving the user space is a bit higher priority than the server space.

**Comment by PaulWessel** [comment]
Looking at David's README file I read this:

> SRTM15+V2.3 - June 29, 2021
> 
> This V2.3 is a major update of V2.2. Hugh harper and David Sandwell downloaded about 1400 new multibeam cruises from the NEIC site.  These were processed with MB_system to remove outer beams and flatten the rails of the innermost beams.  About 156 of the cruises were very bad or duplicated areas already well charted.  905 of the cruises were kept and went through 5 rounds of editing to remove outliers. 
> 
> In addition, we downloaded 9 composite grids of unique data mainly in the Indian ocean.
> aden.cm		antilles.cm	fiji.cm		guiana.cm	iotj.cm		kerguelen.cm	reunion.cm	saint_paul.cm	swir.cm
> 
> SRTM15+V2.2 - April 1, 2021
> 
> This V2.2 is an update of V2.1. Note that V2.2 was made special for the GEBCO 2021 grid.  It is the same as V2.1 except that
> there were about 1000 small edits to clean up the issues with outer beams, etc.  These were done by Pauline Weatheral , David Sandwell and
> Brook Toxer. In addition two land tiles were replaced. This grid was never distributed to the public.

So OK to detect changes in this README file but then we would learn that 2.2 was a tiny thing not really released to the public so we would skip it.  So it is OK that we have been on 2.1 until now. 2.3 is a big change and we will update files as soon as I can.  I think ideally we would want to

1. Install gmt via git on the gmtserver so that the server has an up-to-date GMT installation
2. Run the data processing and tile preparation scripts on the server instead me doing it here.

**Comment by joa-quim** [comment]
I don't think 41 GB in someone else's dataserver is big. I just don't have them, but that should prevent the duplication.
